### ISSUE TITLE
Create csv file and web page for potential targets for tax filers, drawn from IRS spreadsheets

### ISSUE THREAD

**Issue by donboyd5** [PSLmodels/tax-microdata-benchmarking#20]
@nikhilwoodruff @martinholmer 

I wrote [code in R](https://github.com/donboyd5/taxdata-psl/tree/main/targets_mockup) to:

- Get IRS spreadsheets that contain summary data about tax filers for various years.
- Extract and organize the relevant data
- Save the data with potential targets to a [csv file in this folder](https://github.com/donboyd5/taxdata-psl/tree/main/data)
- Create a [web page](https://boyd-targets-mockup.netlify.app/) that
  - has the code (folded)
  - displays the potential targets in a [Datatable explorer](https://boyd-targets-mockup.netlify.app/prepare.html#sec-show-targets) that allows filtering and sorting
  - has links to the code and to other relevant information

The code is written to minimize the risk of error inherent in extracting bits and pieces from different sections of multiple spreadsheets, and to make it easy and low-risk-of-error to add more targets from individual spreadsheets or to add additional IRS spreadsheets.

The data can be used to create summary and distributional targets for PUF variables for tax filers. The data are a superset of variables we'll want for targeting. Hence I call them "potential" targets. In early stages, we'll only target a small number of values. Later, we may target many more, but still not target everything that is in the csv file.

I expect that eventually we'll translate the potential-targets-getting code to python, although that may occur after Phase 3 of the project. The csv file (after review) may be all we need in Phases 2 and 3. Alternatively, we could move the R project into the PSL project.

**Comment by martinholmer** [comment]
@donboyd5,  Has issue #20 (from early March) been resolved?
If so, you should close #20.
If not, please add to issue #20 that explaining what else needs to be done.

**Comment by martinholmer** [comment]
On July 16, @martinholmer asked this about issue #20:
> @donboyd5, Has issue https://github.com/PSLmodels/tax-microdata-benchmarking/issues/20 (from early March) been resolved?
> If so, you should close https://github.com/PSLmodels/tax-microdata-benchmarking/issues/20.
> If not, please add to issue https://github.com/PSLmodels/tax-microdata-benchmarking/issues/20 that explaining what else needs to be done.

There has been no additions to issue #20 since July 16, so it is being closed.

**Comment by donboyd5** [comment]
@martinholmer Thanks for closing. Once we have a contract in place I'll open a new issue with planned steps for states and CDs.

**Comment by martinholmer** [comment]
@donboyd5 said in closed issue #20:
> Once we have a contract in place I'll open a new issue with planned steps for states and CDs.

Please make the issues action-oriented so that merging a pull request will close the issue.
Any commentary that is not immediately action-oriented should be put in a GitHub Discussion.

**Comment by donboyd5** [comment]
Sounds good.
